br-ext/package/optee_rust_examples_ext/optee_rust_examples_ext.mk

```diff
@@ -15,7 +15,7 @@ export OPTEE_DIR = $(@D)/../../..
 export OPTEE_OS_DIR = $(OPTEE_DIR)/optee_os
 export OPTEE_CLIENT_DIR = $(OPTEE_DIR)/out-br/build/optee_client_ext-1.0
 export OPTEE_CLIENT_INCLUDE = $(OPTEE_CLIENT_DIR)/out/export/usr/include
-export PATH := $(PATH):$(OPTEE_DIR)/toolchains/aarch64/bin
+export PATH = $(shell printenv PATH):$(OPTEE_DIR)/toolchains/aarch64/bin
```

Comment:
Nothing wrong with the fix as such, but is there really a need to spawn a new shell for this? Wouldn't almost what you had work?
```
export PATH := ${PATH}:$(OPTEE_DIR)/toolchains/aarch64/bin
```
Note "curly" and not "parenthesis". The only think I can think if is that make complains about recursively setting the PATH. I think I've seen that under certain circumstances.

Reply:
Why doesn't the original line work? What has changed, I assume that it has worked at some time.

Reply:
Thank you for your comments. The line 18 was modified when upgrading buildroot in #525 , it works fine when running `make buildroot`, but encounters error when running `make buildroot OPTEE_RUST_ENABLE=y`.
```
export PATH := $(PATH):/other/path
```
and
```
export PATH := ${PATH}:/other/path
```
work fine in ordinary Makefile, but not here. I have no idea why this is so, only
```
export PATH = $(shell printenv PATH):/other/path
```
could set the PATH correctly and run
```
make buildroot OPTEE_RUST_ENABLE=y
```
successfully. Could you please give me some suggestions?

Reply:
What about just `$PATH`? I.e. `export PATH = $PATH:/other/path`?